src/lib.rs

```diff
@@ -85,6 +85,7 @@ mod slice_container;
 mod strings;
 mod sum_products;
 mod untyped_array;
+pub mod array_like;
```

Comment:
We usually keep the module structure private and re-export what is relevant for the public API.